src/main/resources/changelog.xml

```diff
@@ -971,5 +971,41 @@
 			</column>
 		</addColumn>
 	</changeSet>
+	
+	<changeSet id="add_issues" author="Aron Zwaan">
+		<createTable tableName="repository_issues">
+			<column name="issue_id" type="bigint" autoIncrement="true">
```

Comment:
Then `autoincrement=true` is not necessary as well, because issue_ids are programatically generated.